### DISCUSSION TITLE
Full OpenSource

### DISCUSSION THREAD

**Discussion by tiagogalvao7** [oidc-wp/openid-connect-generic#519] [Q&A]
It is this plugin full open source? Can I add more claims to the authentication?

**Comment by timnolte** [comment]
The plugin doesn't explicitly control the claims available, that is generally controlled by your IDP. You can use hooks in this plugin to leverage additional claims as needed.

Can you explain in more detail what you are trying to accomplish?

**Reply by tiagogalvao7** [reply]
Can this plugin can be implemented in WooCommerce platform right? Sorry if my questions looks stupid or dumb, but I'm new with the WordPress platform and with OpenID connect.

**Reply by timnolte** [reply]
Are you looking to have WooCommerce customers sign in with social media accounts?

**Reply by tiagogalvao7** [reply]
Hello Tim, sorry for the answer delay. What I pretend is to implement an OP like Keycloak. It's possible right?
